nvmeadm/src/nvmf_subsystem.rs

```diff
@@ -243,15 +243,23 @@ impl Subsystem {
 
     /// Returns the particular subsystem based on the nqn and address.
     // TODO: Optimize this code.
-    pub fn get(host: &str, port: &u16, nqn: &str) -> Result<Subsystem, NvmeError> {
+    pub fn get(
```

Comment:
You can add another function, and have existing get call new function to avoid making this a breaking change

Reply:
not sure, what are the odds of this being breaking? I've changed all callers and will merge this together with control-plane changes.